libmamba/src/download/downloader.cpp

```diff
@@ -268,7 +268,9 @@ namespace mamba::download
         {
             std::stringstream ss;
             ss << "Transfer finalized, status: " << http_status << " [" << effective_url << "] "
-               << size << " bytes";
+               << size << " bytes\n";
+            ss << "This channel is not accessible or is invalid.\n";
```

Comment:
This function doesnt seem to be about an error, so that message would be displayed systematically. Are you sure it's the right spot for this hint? Or is there a missing condition, maybe checking the `http_status`?

Reply:
This function is only used once to get this error message; in fact its logic could be inlined at its unique call site. Should we do this instead?

Reply:
Well the function is named `build_transfer_message` and seems to be potentially used anywhere we would need to display the transfer message? If so then yes moving the hint at the call site would be better.
In general, an error message only makes sense in the code that detected the error.
The alternative, if that function is never used elsewhere, would be to rename it. But I think having the hint where the issue is detected makes more sense.